pages/common/git-merge.md

```diff
@@ -13,3 +13,7 @@
 - Merge a branch and create a merge commit:
 
 `git merge --no-ff {{branch_name}}`
+
+- Abort merge:
```

Comment:
Probably mention that - `Abort a merge in case there are conflicts` ?

Reply:
What do you think, @pxgamer? If you both prefer the longer version, how about "in case _of_ conflicts" to make it slightly shorter?

Reply:
I guess it makes sense to mention that it's if there are conflicts. 👍

Reply:
Thanks, improved in 6c53ccd.